### PR TITLE
Feature/add initial docker infra

### PR DESCRIPTION
**Pull request by drrzmr**

### CODE REVIEW

File: docker/base-images/postgres-server:9.5.6/Dockerfile [hunk 1]
```diff
@@ -0,0 +1 @@
+FROM postgres:9.5.6
```
Comment (jonatasnona) [inline docker/base-images/postgres-server:9.5.6/Dockerfile]: essa é versão do Postgres usada na base de FinOps?

Comment (drrzmr) [reply docker/base-images/postgres-server:9.5.6/Dockerfile]: I think so, but It's needed check this information, for now I guess that is ok we use this version, in future we can change easylly

File: docker/base-images/golang:1.9.2-stretch/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+FROM golang:1.9.2-stretch
+
+ENV GOMETALINTER_VERSION="0960299513738ff031fe418b3fcd4f6badc1a095"
+
+RUN set -eux; \
+	go get github.com/golang/dep/cmd/dep; \
+	\
+	git clone https://github.com/alecthomas/gometalinter.git ${GOPATH}/src/github.com/alecthomas/gometalinter; \
+	cd ${GOPATH}/src/github.com/alecthomas/gometalinter; \
+	git checkout -B build; \
```
Comment (jonatasnona) [inline docker/base-images/golang:1.9.2-stretch/Dockerfile]: i think we can remove this line.
i think that **reset --hard** is enough.

**Comment by jonatasnona** [comment]
can we add pgcli tool into postgres-client image?

File: docker/base-images/golang:1.9.2-stretch/Dockerfile (hunk 2 above)
Comment (drrzmr) [reply docker/base-images/golang:1.9.2-stretch/Dockerfile]: Great!

**Comment by jonatasnona** [comment]
when i try to run docker-compose build i receive this error:

**$ docker-compose build**

**ERROR:** Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a version of "2" (or "2.0") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/

my docker-compose version is:
docker-compose version 1.8.0, build unknown

i'll need to upgrade the version of my docker-compose?

**Comment by drrzmr** [comment]
@jonatasnona for this works on your machine yes, I use **version: '3'** on docker-compose.yml. But for this simple build, this version is not mandatory, we can downgrade it to **version: '2'** if update is a problem for you.
However update is a good option, version 3 has many good features that we can use in future, to do upgrade simple command must works:
```
$ pip3 install --user --upgrade docker-compose
```

**Comment by drrzmr** [comment]
@jonatasnona I update readme with some useful requirements informations

**Comment by drrzmr** [comment]
Closes #1

**Pull request merged**